source/manual/mongodb.html.md

```diff
@@ -5,37 +5,45 @@ layout: manual_layout
 
-The onsite backup machine (backup-1.management) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/)
-runs each night to send encrypted backups to an Amazon S3 bucket.
+The on-site backup machine (`backup-1.management`) pulls the latest backup and stores it on disk. [Duplicity](http://duplicity.nongnu.org/) runs nightly and sends encrypted backups to an AWS S3 bucket.
 
-To restore from this method:
+### Restoring
 
- - Fetch a backup from either the dedicated mount point, the onsite machine or the S3 bucket [using duplicity](restore-from-offsite-backups.html) (to decrypt you may need a password kept in the encrypted hieradata.
+ - Fetch a backup from either the dedicated mount point, the on-site machine, or the S3 bucket [using Duplicity](restore-from-offsite-backups.html) (you may need a password kept in the encrypted [hieradata](https://github.com/alphagov/govuk-secrets)).
  - Unzip the file. This will produce a directory of data.
- - `mongo restore --drop <directory>`
+ - Run the command: `mongo restore --drop <directory>`
 
 ## mongodumps to S3
 
-We also take backups that are sent to an Amazon S3 bucket. The timings are defined by parameters [set in the manifest](https://github.com/alphagov/govuk-puppet/blob/master/modules/mongodb/manifests/s3backup/cron.pp),
-but for important MongoDB clusters these may be taken every 15 minutes. The machines which take the backups are defined in hiera node classes.
-These backups are encrypted using GPG, but the functionality is just a straightforward mongodump.
+We also backup to an AWS S3 bucket. 
 
-To restore the very latest backup, there is a script available to use: `/usr/local/bin/mongodb-restore-s3`. The function essentially grabs the latest backup from the S3 bucket, decrypts and unpacks it, and does a `mongo restore`.
+The timings are defined by parameters [set in the manifest](https://github.com/alphagov/govuk-puppet/blob/master/modules/mongodb/manifests/s3backup/cron.pp), but for important MongoDB clusters these may be taken every 15 minutes. The machines which take the backups are defined in hiera node classes.
+
+These backups are encrypted using GPG, but the functionality is similar to mongodump.
+
+### Restoring
+Use the `/usr/local/bin/mongodb-restore-s3` script available on MongoDB machines which have S3 backup enabled. 
+
+This script grabs the latest backup from the S3 bucket, decrypts and unpacks it, and does a `mongo restore`.
+
+Machines which have enabled S3 backups and contain the script will have `mongodb::backup::s3_backups` set to `true` in their yaml configuration (see [`govuk-puppet`](https://github.com/alphagov/govuk-puppet)).
 
 ### mongodumps via `govuk_env_sync` in AWS
```

Comment:
We need restore instructions for this. Doesn't need to happen in this PR though. I think there might already be a card on the RE GOV.UK "Doing" Trello board for it.

Reply:
Noted 👍